lib/cassanity/executors/cql_rb.rb

```diff
@@ -185,7 +184,7 @@ def call(args = {})
             end
 
             # Select the correct keyspace before executing the CQL query
-            if command != :keyspace_create && (keyspace_name = arguments[:keyspace_name])
+            if !command.to_s['keyspace'] && (keyspace_name = arguments[:keyspace_name])
```

Comment:
I guess a regex is better if we always need to do this for keyspace scoped commands, but I'd probably lean towards explicit rather than automatic. I'd add a constant with an array of commands and check for command inclusion. I'd rather need to add something to the array than have something match the command that shouldn't.